### ISSUE TITLE
[BUG] PersistenceFeatureBase is depended on SqlLite

### ISSUE THREAD

**Issue by CCRalfvandenBurg** [elsa-workflows/elsa-core#6254]
## Description
When defining custom Persistence feature, and using sqlserver, There is an error:
`Skip = "Fix efcore setup first (  System.AggregateException : One or more errors occurred. (Services for database providers **'Microsoft.EntityFrameworkCore.SqlServer', 'Microsoft.EntityFrameworkCore.Sqlite'** have been registered in the service provider. Only a single database provider can be registered in a service provider. If possible, ensure that Entity Framework is managing its service provider by removing the call to 'UseInternalServiceProvider'. Otherwise, consider conditionally registering the database provider, or maintaining one service provider per database provider.))"

**Comment by sfmskywalker** [comment]
I haven't seen this before, even when using the SQL Server provider.
The reason the base persistence feature class depends on Sqlite is for convenience for new users, but if it causes issues we should remove that. But it would be good to try and get this issue reproduced first so we can confirm the fix.

Any chance you could produce a simple project that reproduces the issue?

**Comment by CCRalfvandenBurg** [comment]
Yes, indeed i was planning to create an example project. We "fixed" it by removing the DependOnAttribute in our custom feature.
